op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -0,0 +1,147 @@
+
+	runIsthmusDerivationTest := func(gt *testing.T, testCfg *helpers.TestCfg[any]) {
+		t := actionsHelpers.NewDefaultTesting(gt)
+
+		env := helpers.NewL2FaultProofEnv(t, testCfg, helpers.NewTestParams(), helpers.NewBatcherCfg())
+
+		t.Logf("L2 Genesis Time: %d, IsthmusTime: %d ", env.Sequencer.RollupCfg.Genesis.L2Time, *env.Sequencer.RollupCfg.IsthmusTime)
+
+		sysCfgContract, err := bindings.NewSystemConfig(env.Sd.RollupCfg.L1SystemConfigAddress, env.Miner.EthClient())
+		require.NoError(t, err)
+
+		sysCfgOwner, err := bind.NewKeyedTransactorWithChainID(env.Dp.Secrets.Deployer, env.Sd.RollupCfg.L1ChainID)
+		require.NoError(t, err)
+
+		// Update the operator fee parameters
+		_, err = sysCfgContract.SetOperatorFeeScalars(sysCfgOwner, OperatorFeeScalar, OperatorFeeConstant)
+		require.NoError(t, err)
+
+		env.Miner.ActL1StartBlock(12)(t)
+		env.Miner.ActL1IncludeTx(env.Dp.Addresses.Deployer)(t)
+		env.Miner.ActL1EndBlock(t)
+
+		// sequence L2 blocks, and submit with new batcher
+		env.Sequencer.ActL1HeadSignal(t)
+		env.Sequencer.ActBuildToL1Head(t)
+		env.Batcher.ActSubmitAll(t)
+		env.Miner.ActL1StartBlock(12)(t)
+		env.Miner.ActL1EndBlock(t)
+
+		aliceInitialBalance, err := env.Engine.EthClient().
+			BalanceAt(context.Background(), env.Alice.Address(), nil)
+		require.NoError(t, err)
+
+		env.Sequencer.ActL2StartBlock(t)
+		// Send an L2 tx
+		env.Alice.L2.ActResetTxOpts(t)
+		env.Alice.L2.ActSetTxToAddr(&env.Dp.Addresses.Bob)
+		env.Alice.L2.ActMakeTx(t)
+		env.Engine.ActL2IncludeTx(env.Alice.Address())(t)
+		env.Sequencer.ActL2EndBlock(t)
+
+		receipt := env.Alice.L2.LastTxReceipt(t)
+
+		// Check that the operator fee was applied
+		require.Equal(t, OperatorFeeScalar, uint32(*receipt.OperatorFeeScalar))
+		require.Equal(t, OperatorFeeConstant, *receipt.OperatorFeeConstant)
+
+		l1FeeVaultBalance, err := env.Engine.EthClient().BalanceAt(context.Background(), predeploys.L1FeeVaultAddr, nil)
+		require.NoError(t, err)
+
+		baseFeeVaultBalance, err := env.Engine.EthClient().
+			BalanceAt(context.Background(), predeploys.BaseFeeVaultAddr, nil)
+		require.NoError(t, err)
+
+		sequencerFeeVaultBalance, err := env.Engine.EthClient().BalanceAt(context.Background(), predeploys.SequencerFeeVaultAddr, nil)
+		require.NoError(t, err)
+
+		operatorFeeVaultBalance, err := env.Engine.EthClient().BalanceAt(context.Background(), predeploys.OperatorFeeVaultAddr, nil)
+		require.NoError(t, err)
+
+		aliceFinalBalance, err := env.Engine.EthClient().BalanceAt(context.Background(), env.Alice.Address(), nil)
+		require.NoError(t, err)
+
+		require.True(t, aliceFinalBalance.Cmp(aliceInitialBalance) < 0, "Alice's balance should decrease")
+
+		// Check that the operator fee sent to the vault is correct
+		require.Equal(t,
+			new(big.Int).Add(
+				new(big.Int).Div(
+					new(
+						big.Int,
+					).Mul(new(big.Int).SetUint64(receipt.GasUsed), new(big.Int).SetUint64(uint64(OperatorFeeScalar))),
+					new(big.Int).SetUint64(1e6),
+				),
```

Comment:
It's helpful that the arithmetic operations are on different lines for readability, but the inner formatting is confusing. Maybe something like
```suggestion
				new(big.Int).Div(
					new(big.Int).Mul(
						new(big.Int).SetUint64(receipt.GasUsed),
						new(big.Int).SetUint64(uint64(OperatorFeeScalar)),
					),
					new(big.Int).SetUint64(1e6),
				),
```
is more consistent.

Reply:
Fixed